gallery/webpack.config.js

```diff
@@ -115,6 +116,9 @@ module.exports = function(env = { production: false }) {
       new MiniCssExtractPlugin({
         filename: 'gallery.css'
       }),
+      new webpack.EnvironmentPlugin({
+        'PX_API_KEY': ''
```

Comment:
This sets the value within our code to an empty string unless the `PX_API_KEY` environment variable is set during the build, in which case the env var gets passed in as the value in the code.